lib/smart_answer_flows/simplified-expenses-checker/outcomes/you_can_use_result.govspeak.erb

```diff
@@ -30,35 +56,57 @@
   <% end %>
 
   <% if capital_allowance_claimed %>
-    You can’t use simplified expenses for your vehicle because you’ve already claimed Capital Allowances for it.
+    You can’t use simplified expenses for your vehicle because you’ve already claimed capital allowances for it.
 
     You have to continue to calculate your vehicle expenses by working out the actual costs.
   <% end %>
 
   ###What to do for your tax return
 
-  Use the simplified expenses flat rates to work out your business expenses.
+  Use the [simplified expenses flat rates](/simpler-income-tax-simplified-expenses/vehicles-) to work out your business expenses.
 
-  Multiply the flat rates with the [miles you drove](/simpler-income-tax-simplified-expenses/vehicles-), [hours you work from home](/simpler-income-tax-simplified-expenses/business-use-of-your-home) or the [number of people who live on your business premises](/simpler-income-tax-simplified-expenses/private-use-of-business-premises).
+  Multiply the flat rates with either:
 
-  [Self Assessment tax returns](/self-assessment-tax-returns)
+  [the miles you drove](/simpler-income-tax-simplified-expenses/vehicles-)
+  [the hours you worked from home](/simpler-income-tax-simplified-expenses/working-from-home)
+  [the number of people who lived on your business premises](/simpler-income-tax-simplified-expenses/living-at-your-business-premises)
+
+  ##If you work out the actual costs
 
-  ##Calculating expenses by working out the actual costs
+  You could claim:
 
   <% unless vehicle_costs.to_f == 0.0 %>
-    - <%= format_money(vehicle_costs) %> for cars and vans and motorcycles
+    - <%= format_money(vehicle_costs.to_f) %> for the running costs of your car, van or motorcycle
+  <% end %>
+
+  <% unless capital_allowances_estimate.to_f == 0.0 %>
+    - <%= format_money(capital_allowances_estimate.to_f) %> of [capital allowances](/capital-allowances) for your car, van or motorcycle
+  <% end %>
+
+  <% unless home_costs.to_f == 0.0 %>
+    - <%= format_money(home_costs) %> for working from home
+  <% end %>
+
+  <% unless simple_business_costs.to_f == 0.0 %>
+    - <%= format_money(business_premises_cost) %> to deduct from your total business costs - you then claim the balance
   <% end %>
 
   <% unless green_vehicle_write_off.to_f == 0.0 %>
-    - <%= format_money(green_vehicle_write_off) %> for cars, vans and motorcycles
+    <% unless expenses_or_allowances == "no" %>
+      - <%= format_money(green_vehicle_write_off) %> for cars, vans and motorcycles
+    <% end %>
   <% end %>
 
   <% unless dirty_vehicle_write_off.to_f == 0.0 %>
-    - <%= format_money(dirty_vehicle_write_off) %> for for cars, vans and motorcycles
+    <% unless expenses_or_allowances == "no" %>
```

Comment:
I think it would read better if these where `if` statements instead of nested `unless` statements.